From 8bc172170c53eda259d56a98a01c94a146be2c71 Mon Sep 17 00:00:00 2001
From: Zain Wang <zain.wang@rock-chips.com>
Date: Tue, 17 Nov 2015 12:00:45 +0800
Subject: [PATCH] UPSTREAM: clk: rockchip: add an id for rk3288 crypto clk

Add an id for crypto clk to the binding header, so that it can be called
in other part.

Change-Id: I541f4373cb2753aa74e2183cae82215e31faae44
Signed-off-by: Zain Wang <zain.wang@rock-chips.com>
Acked-by: Michael Turquette <mturquette@baylibre.com>
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
(cherry picked from commit 94d5d6a0fbf33c5fde246b61b8c358cfeeba633f)
---
 include/dt-bindings/clock/rk3288-cru.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/rk3288-cru.h b/include/dt-bindings/clock/rk3288-cru.h
index c719aacef14f..9e0a5e9acee9 100644
--- a/include/dt-bindings/clock/rk3288-cru.h
+++ b/include/dt-bindings/clock/rk3288-cru.h
@@ -86,6 +86,7 @@
 #define SCLK_USBPHY480M_SRC	122
 #define SCLK_PVTM_CORE		123
 #define SCLK_PVTM_GPU		124
+#define SCLK_CRYPTO		125
 
 #define SCLK_MAC		151
 #define SCLK_MACREF_OUT		152
-- 
2.35.3

